scripts/_template_script.py

```diff
@@ -10,18 +10,39 @@ def main():
     print(f"You are using the '{network.show_active()}' network")
     deployer = accounts.load("deployer") # prompts for password
     multisig = ApeSafe(MULTISIG_ADDRESSES[CHAIN_IDS["MAINNET"]])
-
+   
     """
-
+   
     Build your txn here
-
+   
     """
 
     # combine history into multisend txn
+    # TODO: set nonce 
     safe_tx = multisig.multisend_from_receipts()
+    safe_nonce = 0
+    
+    # check if nonce is invalid or already in use
+    current_nonce = multisig.contract(multisig.address).nonce()
```

Comment:
Since we know which ABI we expect for this, we should use `Contract.from_abi()` w/ a tiny ABI with just `nonce()` to prevent downloading contract data from etherscan.

Reply:
```python
    nonce_abi = [{
        "stateMutability": "view",
        "type": "function",
        "name": "nonce",
        "inputs": [],
        "outputs": [
                {
                    "name": "",
                    "type": "uint256"
                }
        ]
    }]
    current_nonce = Contract.from_abi("GnosisSafeProxy", multisig.address, nonce_abi).nonce()
```